src/providers/lazy-loading/paramsToQuery.ts

```diff
@@ -11,7 +11,7 @@ interface ParamsToQueryOptions {
 const defaultParamsToQueryOptions = {
   filters: true,
   sort: true,
-  pagination: true,
+  pagination: true
```

Comment:
I personally prefer "trailing commas" (makes merging and rearranging easier) 

Are you able to put that into the `tslint.json` too? Not sure which rule changes it though 🤔 
```
"trailing-comma": ...
"comma-dangle": ...
```